lib/stellar-core-backup/job.rb

```diff
@@ -81,11 +88,18 @@ def run()
             # using sudo, if running as non root uid then you will need to configure sudoers
             stop_core = @cmd.run_and_capture('sudo', ['/bin/systemctl', 'stop', 'stellar-core'])
             # only proceed if core is stopped and FS is clean
-            if stop_core.success then
-              @backup_archive = @s3.get(@s3.latest)
+            if stop_core.success
+              if @clean
+                `rm -fr /var/lib/stellar/buckets/*`
```

Comment:
Were you going to use the `Cmd` class to run all system commands? Using the `Cmd` class will ensure the script output is wholly managed by us instead of interspersing command output with formatted output.

I also think it might be worth parameterising the `bucket_dir` ?

Reply:
Where there are stray unmanaged `calls`, they will indeed be managed by the Cmd class.

Reply:
Ok cool. I think it should be a configuration option set in the config file so it can be overridden

Reply:
This is already implemented in the code. The bucket directory is configured in the stellar-core.cfg file and read in as the @fs_restore.core_data_dir var. If there is a use case to override this and restore to a completely foreign target directory then we can implement that, otherwise this is resolved.

```grep BUCKET_DIR_PATH /etc/stellar/stellar-core.cfg
BUCKET_DIR_PATH="/var/lib/stellar/buckets"```

Reply:
Nice!